scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -756,22 +757,22 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 {% macro dismissed_panel() %}
   <div class="card border-danger mb-3" style="border-width: 5px; display: block;">
     <div class="card-header bg-danger text-black">
-      <a name="dismissed_variants"><strong>Dismissed Variants</strong></a>
+      <p id="dismissedDesc"><a id="dismissed_variants"><strong>Dismissed Variants</strong></a></p>
     </div>
     <div class="card-body">
       {% if variants.dismissed_detailed %}
-        <table class="table table-striped table-sm" style="background-color: transparent">
+        <table class="table table-sm" style="background-color: transparent" aria-describedby="dismissedDesc">
           <thead>
             <tr>
-                <td width="5%"></td>
-                <td width="25%"><strong>Variant</strong></td>
-                <td width="5%"><strong>Category</strong></td>
-  	            <td width="25%"><strong>Genes</strong></td>
-                <td width="45%"><strong>Dismissed description</strong></td>
+                <th class="w-5"></th>
+                <th class="w-25"><strong>Variant</strong></th>
+                <th class="w-5"><strong>Category</strong></th>
+  	            <th class="w-25"><strong>Genes</strong></th>
+                <th class="w-45"><strong>Dismissed description</strong></th>
             </tr>
           </thead>
           {% for variant in variants.dismissed_detailed|sort(attribute='variant_rank') %}
-            <tr>
+            <tr{% if loop.index0 % 2 %} style="background-color: #d1d5db;" {% endif %}>
```

Comment:
Nice touch, good to differentiate the entries! Would appreciate a css-defined color! 😊